cmd/csi-proxy/main.go

```diff
@@ -34,11 +36,12 @@ func (i *workingDirFlags) Set(value string) error {
 }
 
 var (
-	kubeletPath    = flag.String("kubelet-path", `C:\var\lib\kubelet`, "Prefix path of the kubelet directory in the host file system")
-	windowsSvc     = flag.Bool("windows-service", false, "Configure as a Windows Service")
-	requirePrivacy = flag.Bool("require-privacy", true, "If true, New-SmbGlobalMapping will be called with -RequirePrivacy $true")
-	service        *handler
-	workingDirs    workingDirFlags
+	kubeletPath     = flag.String("kubelet-path", `C:\var\lib\kubelet`, "Prefix path of the kubelet directory in the host file system")
+	windowsSvc      = flag.Bool("windows-service", false, "Configure as a Windows Service")
+	requirePrivacy  = flag.Bool("require-privacy", true, "If true, New-SmbGlobalMapping will be called with -RequirePrivacy $true")
+	metricsBindAddr = flag.String("metrics-bind-addr", "localhost:10010", "The address the metric endpoint binds to.")
```

Comment:
I see that kube-proxy uses `metrics-bind-address`, let's use that as well. 

https://github.com/kubernetes/kubernetes/blob/dc3021b156a6a5adc4b4898d366822529bcc74a4/cmd/kube-proxy/app/options.go#L119